lib/helpers/getOneBlockPerCondition.m

```diff
@@ -0,0 +1,33 @@
+% Want a function which, given an input argument of the block number,
+% returns the condition of that block (medical or monetary) and the block's
```

Comment:
Sorry, just noticed this — this does not describe the function at all. (Or rather, it describes lines 23 and 24 of the function.)